win32ss/gdi/ntgdi/freetype.c

```diff
@@ -2383,7 +2383,7 @@ FontFamilyInclude(LPLOGFONTW LogFont, PUNICODE_STRING FaceName,
 
 static BOOL FASTCALL
 FontFamilyFound(PFONTFAMILYINFO InfoEntry,
-                PFONTFAMILYINFO Info, DWORD InfoCount, WCHAR *pszSubstName)
+                PFONTFAMILYINFO Info, DWORD InfoCount, LPCWSTR pszSubstName)
```

Comment:
PCWSTR, instead of LPCWSTR !!

Reply:
Huh? There's no difference.

Reply:
Correct, except that you see LPxxx in 99.99999% of user-mode win32 code (and 0.00001% in kernel-mode code) (because it's an anachronism).

Reply:
That's not true for win32k

Reply:
Done.